Sources/Files.swift

```diff
@@ -197,7 +197,13 @@ public class FileSystem {
 
         /// The date when the item was last modified
         public private(set) lazy var modificationDate: Date = self.loadModificationDate()
-
+        
+        /// The date when the item was created, or `nil` if the information is not available
+        public private(set) lazy var creationDate: Date? = self.loadCreationDate()
+        
+        /// The size of the item in bytes, or `nil` if the information is not available
+        public private(set) lazy var size: UInt64? = self.loadSize()
```

Comment:
It would be preferred to expose `size` as an `Int` (as opposed to `UInt64`).

Reply:
Also, of note, @JohnSundell's [most recent article](https://www.swiftbysundell.com/posts/writing-self-documenting-swift-code) suggests that `size` could be more "self-documenting" by creating a `typealias` for `Int` called `ByteCount` to explicitly state that `size` is in terms of bytes.

Reply:
Thanks @clayellis Great advice!